### DISCUSSION TITLE
Kurento Capacity Details!!!

### DISCUSSION THREAD

**Discussion by Sand2407** [Kurento/kurento#71] [Community]
Dear Members,

Over the past few days, I have been searching for materials that provide insights into the capacity details supported by the "**Kurento Media Server**". Unfortunately, I haven't been able to find any solid figures. I understand that the actual capacity depends on various factors such as my load test environment and the scenarios I am testing etc. However, based on the following scenarios, could anyone guide me with approximate figures from their load test environments?

One-to-One WebRTC video call
WebRTC to VoIP one-to-one call

Thank you for your assistance.

Best regards,

**Comment by a-sure** [comment]
On GCP 'generic' machine type, we had approx 9-10 one-to-one calls, i.e. 18-20 participants per Kurento core.
